benchmarks/benchmark_functions/default_settings.py

```diff
@@ -99,11 +91,9 @@ def _vqe_defaults(hyperparams):
 
     options_dict = {"interface": interface, "diff_method": diff_method, "optimize": optimize}
 
+    # if device name is given, create device
     if isinstance(device, str):
-        if device == "cirq.pasqal":
-            device = qml.device(device, wires=n_wires, control_radius=1.5)
-        else:
-            device = qml.device(device, wires=n_wires)
+        device = qml.device(device, wires=len(hf_state), shots=None)
```

Comment:
Would that not mean we cannot run on the pasqal device (at least not when passing a string)?

Reply:
Given this is not part of the device benchmark suite, it'll probably be fine.